### DISCUSSION TITLE
1.19.4中打开容器的协议又修改了？

### DISCUSSION THREAD

**Discussion by 223225zzzkkk** [Tnze/go-mc#246] [Q&A] [answered]
#147

**Comment by Tnze** [comment, marked as answer]
Maybe <https://wiki.vg/Protocol#Use_Item_On> ?

**Reply by 223225zzzkkk** [reply]
谢谢，这是代码`client.Conn.WritePacket(pk.Marshal(0x31,
 pk.VarInt(0), 
pk.Position{0, 63, 6}, 
pk.VarInt(0),
 pk.Float(0.5), 
pk.Float(0.5), 
pk.Float(0.5),
 pk.Boolean(true), 
pk.VarInt(0)))`
